### ISSUE TITLE
Suggest: Add roadmap to repo (currently contains world-editable garbage)

### ISSUE THREAD

**Issue by sc0ttwad3** [mozilla/Fira#213]
### Roadmap document contains world-editable Garbage

The linked Roadmap document is editable with full permissions to any/everyone that clicks the link.  It's not in the repo and continually is full of new/edited garbage. I'd suggest including this document in the repo and require a pull-request with any update/changes, once a real current Roadmap document is made.

**Comment by chdsbd** [comment]
You're mistaken. What your are seeing is the "suggesting" view. If you switch to "viewing" mode, you can see that there isn't any vandalism.

**Comment by sc0ttwad3** [comment]
Thanks, the roadmap is no longer writeable.  I know about the suggesting view/viewing mode... What was present in the document--when it was writeable--was the typical juvenilia you see on bathroom stall walls, gibberish, and the unimaginative  "jjjjjjjjkkkkkkklllllll" type key presses, then my probative "test, test, test".

**Comment by sc0ttwad3** [comment]
Thanks, the roadmap is no longer writeable.  I know about the suggesting view/viewing mode... What was present in the document--when it was writeable--was the typical juvenilia you see on bathroom stall walls, gibberish, and the unimaginative  "jjjjjjjjkkkkkkklllllll" type key presses, then my probative "test, test, test".

**Comment by sc0ttwad3** [comment]
I spoke too soon and misunderstood. I guess when in the suggesting view, the juvenilia still shows from other users?  That's who I'm again seeing all the "piece of sh#t," "f$&k you" type commentary inserted within the document?
